pkg/operator/controller.go

```diff
@@ -787,7 +790,7 @@ func (c *Controller) syncMachineConfig(pools []*mcfgv1.MachineConfigPool, labels
 
 	bootcmdline, bootcmdlineSet := c.pc.state.bootcmdline[profile.Name]
 	if !bootcmdlineSet {
-		klog.V(2).Infof("syncMachineConfig(): bootcmdline for %s not cached, sync cancelled", profile.Name)
+		klog.V(2).Infof("syncMachineConfig(): bootcmdline for %s not cached, sync canceled", profile.Name)
```

Comment:
That will take a lot of getting used to.  US English... Sigh.

Reply:
There are options.. 
I have configured the locale to `US` but it can be left as `default` (empty) and `golangci-lint` will use a "neutral variety of English" [see](https://golangci-lint.run/usage/linters/#misspell)

Also we can add the most used british words to the `ignore-words` list.